.github/pr-blocking-todos/checkfortodos.py

```diff
@@ -0,0 +1,64 @@
+import sys
+
+def checkForTodos(fnameToCheck):
+    """Check for file containing todos
+
+    If there are TODOs found, print them
+
+    Return the exit code to return to the shell
+
+    0 -- No TODOs; all is well
+    1 -- TODOs found
+    2 -- TODO file not found
+    3 -- No/ invalid TODO file name specified
+    """
+
+    ## ensure fnameToCheck is valid
+    if ( fnameToCheck is None 
+           or not isinstance(fnameToCheck, str) 
+           or len(fnameToCheck) < 1
+        ):
+        return 3
+
+    try:
+        ## attempt to open file
+        with open(fnameToCheck, 'r') as f:
+            for line in f:
+                if len(line) > 0 and line[0] == '*':
```

Comment:
* [ ] This check ought to be refactored into a method -- something like `isTodoLine`.
* [ ] Other types of markdown bullet markers should be allowed -- e.g., `-`
* [ ] This script should better document which lines are considered TODOs (though it can point to the `isTodoLine` function documentation
* [ ] One should consider (and certainly document) whether white space at the start of a line precludes it from being a TODO line.  Currently it does not, and I think that's fine.